solvebio/resource/object.py

```diff
@@ -526,3 +528,10 @@ def tag(self, tags, remove=False, dry_run=False, apply_save=False):
             self.save()
 
         return True
+
+    def untag(self, tags, dry_run=False, apply_save=False):
+        """Remove tags on an object"""
+
+        untag_part = partial(self.tag, remove=True)
+
+        return untag_part(tags=tags, dry_run=dry_run, apply_save=apply_save)
```

Comment:
Is partial needed? Why not:

```
        return self.tag(tags=tags, remove=True, dry_run=dry_run, apply_save=apply_save)
```

Reply:
the habit...it is not needed. Will change to the suggested.